app/views/pages/_top_works_list.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2 class='top-ten-header'><%= list_name %></h2>
+<ul class='top-ten'>
+
+  <% model.first(10).each do |work| %>
```

Comment:
Good work using a partial to DRY this up. However, I would probably not call `.first(10)` here, but rather in the controller (or better a model method).